client/src/views/workspaces/WorkspacesPage.vue

```diff
@@ -362,10 +363,28 @@ async function openWorkspaceContextMenu(event: Event, workspace: WorkspaceInfo):
       onWorkspaceShareClick(new Event('ignored'), workspace);
     } else if (data.action === WorkspaceAction.CopyLink) {
       await copyLinkToClipboard(workspace);
+    } else if (data.action === WorkspaceAction.OpenInExplorer) {
+      await openWorkspace(workspace);
     }
   }
 }
 
+async function openWorkspace(workspace: WorkspaceInfo): Promise<void> {
+  const result = await getSystemPath(workspace.handle, '/');
+
+  if (!result.ok) {
+    await informationManager.present(
+      new Information({
+        message: translate('FoldersPage.open.folderFailed', { name: workspace.currentName }),
+        level: InformationLevel.Error,
+      }),
+      PresentationMode.Modal,
+    );
+  } else {
+    window.electronAPI.openFile(result.value);
```

Comment:
I've hidden the action on web so it can't be selected by the user, and `window.electronAPI` is mocked on web so at worst it just does nothing, but it won't cause an error.

Reply:
lgtm